paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -208,12 +212,26 @@ def parse_args() -> argparse.Namespace:
         default="mesos",
         choices=["mesos", "kubernetes"],
     )
+    parser.add_argument(
+        "--namespace-prefix",
+        help="prefix of the namespace to fetch the logs for",
+        dest="namespace_prefix",
+        default="paasta",
+    )
     return parser.parse_args()
 
 
 def main(args: argparse.Namespace) -> None:
     cluster = load_system_paasta_config().get_cluster()
-    info_list = get_task_allocation_info(args.scheduler)
+    client = KubeClient()
+    all_namespaces = kubernetes_tools.get_all_namespaces(client)
+    matching_namespaces = [n for n in all_namespaces if args.namespace_prefix in n]
+    for matching_namespace in matching_namespaces:
```

Comment:
This seems like it will duplicate data for Mesos - e.g. if there are 3 matching kubernetes prefixes, we'll output data for Mesos 3 times. Can you apply the namespace logic only if the scheduler is Kubernetes?

Reply:
Good catch. Addressed.